### PR TITLE
EA: Frequency must always have a Value

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#66515](https://central.tri.be/issues/66515)

### CODE REVIEW

File: src/resources/js/aggregator-fields.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ tribe_aggregator.fields = {
 		dropdown                : '.tribe-ea-dropdown',
 		origin_field            : '#tribe-ea-field-origin',
 		import_type_field       : '.tribe-import-type',
+		import_type_field       : '.tribe-import-type',
```
Comment (barryhughes) [inline src/resources/js/aggregator-fields.js]: Duplicate (of line 13)?

File: src/resources/js/aggregator-fields.js [hunk 2]
```diff
@@ -100,7 +101,16 @@ tribe_aggregator.fields = {
...
 			.on( 'blur'       , obj.selector.datepicker                , obj.date_helper )
 			.on( 'submit'     , obj.selector.tab_new                   , obj.events.suppress_submission )
-			.on( 'change'     , obj.selector.import_type_field         , obj.reset_preview )
+			.on( 'change'     , obj.selector.import_type_field         , function() {
+				// Resets the Preview
+				obj.reset_preview()
+
+				// Every time you change Type of import we reset the frequency field
+				var $this = $( this ),
+					$frequency = $( this ).next( obj.selector.fields );
```
Comment (barryhughes) [inline src/resources/js/aggregator-fields.js]: Indent using the same number of tabs as the previous line, then spaces to align `$frequency` with `$this`?

**Comment by bordoni** [comment]
Ready for another look

File: src/resources/js/aggregator-fields.js [hunk 3]
```diff
@@ -106,7 +105,7 @@ tribe_aggregator.fields = {
 				obj.reset_preview()
 
 				// Every time you change Type of import we reset the frequency field
-				var $this = $( this ),
+				var	$this = $( this ),
```
Comment (barryhughes) [inline src/resources/js/aggregator-fields.js]: Shouldn't it be done with spaces though? I feel like our coding guidelines say as much (ie, initial indent with tabs, subsequent alignment with spaces):

```
> > > var $something,
> > >     $other;
```

(Where `>` is a tab)

Comment (bordoni) [inline src/resources/js/aggregator-fields.js]: Ohhh ok. Sounds good.

**Comment by bordoni** [comment]
Ready for another look

**Pull request merged**